Uyanda.Coffee.Application/Features/BeverageManagement/Services/IBeverageManagementService.cs

```diff
@@ -14,5 +14,9 @@ public interface IBeverageManagementService
         Task<PurchaseResult> PurchaseAsync(PurchaseCommand purchase);
 
         Task<UpsertBeverageSizeCostResult> UpsertBeverageSizeCostAsync(UpsertBeverageSizeCostCommand costSize);
+
+        Task<CustomerPurchaseResult> UpsertCustomerPurchaseAsync(CustomerPurchaseCommand purchase);
+
+        Task<RedeemPointsPurchaseResult> PurchaseRedeemPointsAsync(RedeemPointsPurchaseCommand purchase);
```

Comment:
You would rather keep on method that handles purchases. So, keep the _PurchaseAsync_ method only, and then have the _PurchaseCommand_ class also contain information like:
1. Number of coffee cups a user is settling using points